TMaps for learning STS cardiac surgery outcomes from Partners ECGs

erikr opened the pull request:
This PR implements TMaps for the Society of Thoracic Surgeons (STS) cohort and their outcomes. It closely follows @lucidtronix's existing approach for Loyalty TMaps.

Input: 12L Partners ECGs

Labels: eight outcomes:
1. death within 30 days after surgery
2. Prolonged ventilation
3. Stroke
4. Deep sternal wound infection
5. Prolonged length of stay (need to clarify how this was defined)
6. Renal failure
7. Re-operation
8. Any of above outcomes.

I'd like to make this PR while it is small and close to `master`.

Preliminary results:
![rocs_together](https://user-images.githubusercontent.com/1253422/78210718-06131480-7479-11ea-812b-cbeefabc1a03.png)

Commit:
adds tmaps for STS outcomes

Commit:
fixes error finding ECGs within preop window

Commit:
run script specifies GPU; transfer learning via --model_layers

Commit:
revises shell script to do transfer learning

lucidtronix reviewed (approved):
@erikr Looks great! A refactor request, and then good to merge. no need to bounce back.

Commit:
replaces sts with more descriptive name, cardiac_surgery

Commit:
Merge branch 'master' into er_sts_ecg

Commit:
Removes sh scripts, renames sts to cardiac_surgery

1. Removes *.sh run scripts from root dir of ml repo.
2. Restores arguments.py to master.
3. Renames sts with more descriptive cardiac_surgery.

Commit:
deletes backup version of tensor_maps script that was ravaged by Black
